types/globals.d.ts

```diff
@@ -3,15 +3,17 @@ import { NightwatchAPI } from './index';
 /**
```

Comment:
This commit looks good overall, however, there are a few areas where improvements can be made:

1. **Missing JSDoc Comments:** Some interfaces and properties are not documented. For example, the `NightwatchAPI` interface is not documented. It would be beneficial to add JSDoc comments to all interfaces and properties for better understanding and maintainability.

   ```typescript
   /**
    * Interface for Nightwatch API
    */
   export interface NightwatchAPI { ... }
   ```

2. **Inconsistent Example Usage:** The examples provided in the JSDoc comments are inconsistent. Some properties have examples while others don't. It would be better to provide examples for all properties for consistency and better understanding. 

   ```typescript
   /**
    * @example
    * asyncHookTimeout: 10000
    */
   asyncHookTimeout?: number;
   ```

3. **Use of `unknown` Type:** The use of `unknown` type in some function parameters might be too broad and could lead to potential type errors. If possible, provide more specific types.

   ```typescript
   reporter?(results: unknown, done: (...args: unknown[]) => void): void;
   ```

4. **Optional Properties:** Some properties are optional. If these properties are essential for the functioning of the interface, consider making them mandatory.

   ```typescript
   /**
    * Controls the timeout value for async hooks.
    *
    * @default 10000
    *
    * @example
    * asyncHookTimeout: 10000
    */
   asyncHookTimeout?: number;
   ```

Please consider these changes to improve the quality of the code.